### DISCUSSION TITLE
Suggest support Visual Studio 2019/2022

### DISCUSSION THREAD

**Discussion by dameng324** [community/community#8579] [Copilot Conversations] [answered]
Hi!
I am a C# programmer. Github copilot is amazing, but I cannot use it in Visual Studio.

I have seen that github copilot is supported in Jetbrains Idea and Pycharm.

Do you have any plan that support Visual Studio 2019 or 2022?

**Comment by kzu** [comment, marked as answer]
There is a fix available now: [Copilot for Visual Studio 2022](https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs) 😉
